### PR TITLE
html tamplate

### PR DESCRIPTION
**Pull request by Snizhana-Uzumaki**

### CODE REVIEW

**Comment by Snizhana-Uzumaki** [comment]
Finish task

**Comment by Snizhana-Uzumaki** [comment]
Fix padding

File: home_page.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                <span class="profile-list-icon"><iconify-icon icon="mdi:menu-down-outline"></iconify-icon></span>
+            </a>
+        </div>
+    </header>
+
+    <section class="ad-info">
+        <ul class="ad-info-list">
+            <li class="ad-info-block odd">
+                <div class="ad-info-block-text">
+                    <p class="meaning-platform">E-COURSE PLATFORM</p>
```
Comment (Bezart) [inline home_page.html:30]: Currently "Montserrat" font family is not applied to this paragraph as well as font-weight

File: home_page.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>global-talk-templat</title>
+    <script src="https://code.iconify.design/iconify-icon/1.0.3/iconify-icon.min.js"></script>
+</head>
+<body>
+    <header class="site-navigation">
```
Comment (Bezart) [inline home_page.html:12]: The header height is much larger than in the mock-up

File: style.css [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+            --body-two: 600 16px/22px 'Montserrat';
+            --body-three: 600 14px/20px 'Montserrat';
+            --body-four: 600 12px/17px 'Montserrat';
+            --body-five: 600 14px/140% 'Montserrat';
+            --body-six: 600 12px/140% 'Montserrat';
+            --body-seven: 600 12px/100% 'Montserrat';
+            --body-eight: 600 10px/100% 'Montserrat';
+            --body-nine: 600 16px/100% 'Montserrat';
+        
+        /*button*/
```
Comment (Bezart) [inline style.css:66]: Check the values of these variables. You misspelled "px" and  it breaks a lot of elements on the page

File: home_page.html [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+        <div class="user-account">
+            <a class="user-account-link" href="">
+                <img class="avatar" src="user_avatar.svg" alt="user avatar">
+                <span class="user-name">Alex</span>
+                <span class="profile-list-icon"><iconify-icon icon="mdi:menu-down-outline"></iconify-icon></span>
+            </a>
+        </div>
+    </header>
+
+    <section class="ad-info">
```
Comment (Bezart) [inline home_page.html]: What is "ad"? This term is usually means "advertisement" and indicates content that is not part of the website but rather third party promotional blocks. I would recommend naming this block differently

File: home_page.html [hunk 5]
```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>global-talk-templat</title>
```
Comment (Bezart) [inline home_page.html]: Please use the "readable" title, it should be clear  for users what site they are looking at.

File: home_page.html [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+                <img class="ad-info-block-img-two" src="girl_with_table.svg" alt="girl with table">
+                <div class="ad-info-block-text">
+                    <h1 class="block-title">Learn a language in a playful way</h1>
+                    <p class="block-description">Make learning words more fun with mini-games</p>
+                </div> 
+                <div class="ad-info-block-game-points-conteiner">
+                    <div class="ad-info-block-game-points">
+                        <a class="sprint"  href="">
+                            <img class="sprint-img" src="sneakers.svg" alt="sneakers">
+                            <p class="sprint-name" >Sprint →</p>
```
Comment (Bezart) [inline home_page.html]: For arrows, don't just insert symbol, but use an html character code instead

File: style.css [hunk 7]
```diff
@@ -0,0 +1,356 @@
...
+            --link-one: 16px/20px 'Montserrat';
+            --link-two: 14px/20px 'Montserrat';
+    /*Padding*/
+        --padding-one:40px 20px 0 20px;
+        --padding-two:12px 20px;
+        --padding-three: 0 0 40px 0;
+        --padding-four: 18px 20px;
+        --padding-five: 4% 0 0 0;
+        --padding-six: 20px 0 40px 0;
+        --padding-seven: 84px 30.5px 14px 30.5px;
```
Comment (Bezart) [inline style.css]: Never use half pixel values. If you see this in the mock-up, it is a mistake

File: style.css [hunk 8]
```diff
@@ -0,0 +1,356 @@
...
+        /*button*/
+            --button-one: 700 18px/25px 'Montserrat';
+            --button-two: 700 16x/22px 'Montserrat';
+            --button-three: 700 12x/100% 'Montserrat';
+            --button-four: 700 14x/100% 'Montserrat';
+
+        /*links*/
+            --link-one: 16px/20px 'Montserrat';
+            --link-two: 14px/20px 'Montserrat';
+    /*Padding*/
```
Comment (Bezart) [inline style.css:75]: It is not the best way to define paddings variables. Try to define the scale order, e.g.:

spacing-xxs: 2px
spacing-xs: 4px
spacing-s: 8px
spacing-m: 16px
....

Than use those tokens separately
padding: spacing-s
padding-right: spacing-m
...
etc

Not every padding value should be a variable

**Review by Bezart — CHANGES_REQUESTED** [review]
@Snizhana-Uzumaki please check my comments and read the guide how to add descriptions to pull requests https://infinum.com/blog/write-good-pull-requests/

**Pull request closed without merging**